terraform/projects/app-ecs-services/config/alerts/registers.yml

```diff
@@ -9,3 +9,22 @@ groups:
     annotations:
         summary: "App {{ $labels.app }} has too many 5xx errors"
         description: "App {{ $labels.app }} has 5xx errors in excess of 25% of total requests"
+
+  - alert: Registers_DiskSpaceExceeded
+    expr: disk_utilization{org="openregister",space="prod"} > 80
+    for: 300s
+    labels:
+        product: "registers"
+    annotations:
+        summary: "App {{ $labels.app }} has exceeded 80% disk space usage for 5 minutes"
+        description: "TODO"
+
+  - alert: Registers_CPUExceeded
+    expr: avg_over_time(cpu{org="openregister",app="beta-multi",space="prod"}[5m])
```

Comment:
I'm happy that `avg_over_time` is indeed the moving average we were after based on https://github.com/prometheus/prometheus/issues/383

On another note, if you play around in the prometheus interface and up the range you are averaging over (currently 5m), the graph does indeed become much smoother as we expected. Changing the range to 30s (which is currently how often new metrics are being scraped) gives you the identical graph to it without using `avg_over_time`. It may be worth us playing with this value for your graphs. We could also expand the summary to make it clear this is based on a moving average being above 80% for 5 minutes rather than every single data point being above 80% for the last 5 minutes.

Next step after that is also to look at removing the restriction to a single app but to include all your apps.